libs/sdk/eslint-schematics/package.json

```diff
@@ -24,12 +24,9 @@
     "packageGroupName": "@skyux-sdk/eslint-schematics",
     "migrations": "./migrations.json",
     "packageGroup": {
-      "@eslint/js": "^9.26.0",
       "@skyux-sdk/eslint-schematics": "0.0.0-PLACEHOLDER",
       "eslint-config-skyux": "0.0.0-PLACEHOLDER",
-      "eslint": "^9.24.0",
-      "skyux-eslint": "0.0.0-PLACEHOLDER",
-      "typescript-eslint": "^8.32.0"
```

Comment:
Public facing consumers should be able to handle updating these packages as they see fit; I've moved these dependencies to our internal `eslint-config-skyux` package group, instead.